### PR TITLE
Stuster

### PR DESCRIPTION
**Pull request by stuster**

### CODE REVIEW

**Comment by stuster** [comment]
Please review @casey-collab

File: english/dream/dream.md [hunk 1]
```diff
@@ -23,3 +23,5 @@ Afterwards, you decide to:
 [Drink a nice cup of tea](../tea/tea.md)
 
 [swim](../swim/swim.md)
+
+[become a tender defender](../tender-defender/tender-defender.md)
```
Comment (casey-collab) [inline english/dream/dream.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/dream/tender-defender/tender-defender.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+A woman benefits from a kind, tender partner who cares for her and a strong, decisive protector who can defend her.
```
Comment (casey-collab) [inline english/dream/tender-defender/tender-defender.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by stuster** [comment]
Please review @casey-collab

File: english/dream/dream.md (hunk 1 above)
Comment (casey-collab) [inline english/dream/dream.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/dream/tender-defender/tender-defender.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+DON'T BE: selfish
+
+DO BE: confident and decisive
+DON'T BE: submissive or passive
+
+DO BE: ready for anything
+DON'T BE: completely oblivious or totally anxious
+
+If you can do all these things, you will be on your way to becoming 
+[what women want](https://www.amazon.com/Mate-Become-Man-Women-Want-ebook/dp/B00RTY0FKK)
```
Comment (casey-collab) [inline english/dream/tender-defender/tender-defender.md]: This link is also invalid.

File: english/dream/tender-defender/tender-defender.md (hunk 2 above)
Comment (casey-collab) [inline english/dream/tender-defender/tender-defender.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by stuster** [comment]
Please review @casey-collab

File: english/dream/tender-defender/tender-defender.md (hunk 3 above)
Comment (casey-collab) [inline english/dream/tender-defender/tender-defender.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/dream/tender-defender/tender-defender.md (hunk 3 above)
Comment (stuster) [inline english/dream/tender-defender/tender-defender.md]: Link is working correctly.

Comment (stuster) [inline english/dream/tender-defender/tender-defender.md]: Please review @casey-collab

**Comment by stuster** [comment]
Please review @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by stuster** [comment]
Please review @casey-collab

**Comment by casey-collab** [comment]
Looks great! :sunglasses:

**Pull request merged**